### DISCUSSION TITLE
Inject data into a wire:ignore element

### DISCUSSION THREAD

**Discussion by mokhosh** [livewire/livewire#1988] [1. Help]
I'm using filepond for image upload, and I have a `model` on my component that changes. I want to show the current image of that `model` inside filepond until something new is uploaded.

Everything is wired correctly, but since I have to use `wire:ignore` on filepond, I can't render `{{ $model->image }}` inside filepond.

Any ideas how to accomplish this?

**Comment by jase-languasco** [comment]
Can you assign it as a property in your component class and then access it like so `@this.modalImage`?

**Reply by mokhosh** [reply]
One problem is accessing the data. Another is to know when it should get the data, because it doesn't update and is not reactive like other `wire:model` elements.

So, what I ended up doing is I dispatch a browser event and pass the image url as a parameter, then catch than event with alpine and set the `src` of the image element. It works, but doesn't feel Livewire-y Alpine-y.

Did you have something different in mind?

**Reply by joshhanley** [reply]
Have you tried using alpine using @entangle? `x-data="{ image: @entangle('model.image') }"` and then setting the src on the image to be bound to `image`?

**Reply by mokhosh** [reply]
`@entangle` is so unreliable in complex views and nested elements that I'm putting a lot of time to refactor and remove it from all of my code.

**Reply by joshhanley** [reply]
Yeah fair. You could still do a similar thing and bind the image src to an Alpine variable, and then use Alpine to get and store the initial image path from Livewire on load. You could also use an Alpine watch function to monitor the livewire property for changes, and then update the Alpine variable with any changes. 

That should save you having to dispatch a browser event from livewire and manually update the src on the image. Will seem more like the Livewire/ Alpine way as you are being more declarative

**Reply by mokhosh** [reply]
> then use Alpine to get and store the initial image path from Livewire on load.

The initial can be set with Blade. `wire:ignore` is only a problem with updates.

> You could also use an Alpine watch function to monitor the livewire property for changes, and then update the Alpine variable with any changes.

This actually crossed my mind, but then I tried `@this.propertyName` and found out the properties are not available like that on `@this`.
